dev/devicelab/manifest.yaml

```diff
@@ -106,6 +106,12 @@ tasks:
     stage: devicelab
     required_agent_capabilities: ["mac/android"]
 
+  tiles_scroll_perf__timeline_summary:
+    description: >
+      Measures the runtime performance of the tiles tab in the Complex Layout sample app on Android.
+    stage: devicelab
+    required_agent_capabilities: ["mac/android"]
```

Comment:
Unless you are testing something Mac-specific I'd use "linux/android". Our Linux agents are much more reliable than Macs.

Reply:
Done. I've also changed complex_layout_perf to linux/android.